examples/css/example.scss

```diff
@@ -37,7 +35,13 @@ body{
     }
     margin: 0 0 40px 0;
     text-align: center;
-
+  }
+  h2 {
+    font-size: 1.2em;
```

Comment:
Properties should be ordered font-family, font-size, font-weight, margin, text-align